### ISSUE TITLE
Empty fetchMyTrades response with bybit testnet

### ISSUE THREAD

**Issue by davidchieregato** [ccxt/ccxt#7091]
- OS: Windows 10
- Programming Language version: NodeJS 12
- CCXT version: 1.29.58

I get an empty result with a fetchMyTrades call, but the api is returning the data as of verbose output. 

Another unrelated small question, for creating an order the parameters `close_on_trigger` and `reduce_only` are mandatory for bybit, is there any reason to not put a default value?

```
const ccxt = require ('ccxt');

var exchange = new ccxt.bybit ({
    'enableRateLimit': true,
    'verbose':true,
    'apiKey': 'xxx',
    'secret': 'xxx',
    'urls': {
        'api': 'https://testnet.bybit.com/'
    }
});
exchange.urls['api'] = exchange.urls['test']
/*exchange.createOrder("BTC/USDT", 'limit', 'buy', 0.01,9000,{"close_on_trigger":false,"reduce_only":false}).then((o)=>{
    console.log(o)
}).catch((e)=>{
    console.log(e)
})*/
exchange.fetchMyTrades("BTC/USDT", 1591635962000, 100).then((o)=>{
    console.log(JSON.stringify(o))
}).catch(function (e) {console.log(e)});
```

```
> node test.js
fetch:
 bybit GET https://api-testnet.bybit.com/v2/public/symbols
Request:
 {}
 undefined

handleRestResponse:
 bybit GET https://api-testnet.bybit.com/v2/public/symbols 200 OK
Response:
 {
  'Cache-Control': 'max-age=0, no-cache, no-store',
  Connection: 'keep-alive',
  'Content-Length': '1904',
  'Content-Type': 'application/json; charset=utf-8',
  Date: 'Tue, 09 Jun 2020 21:01:41 GMT',
  Expires: 'Tue, 09 Jun 2020 21:01:41 GMT',
  Pragma: 'no-cache'
}
 {"ret_code":0,"ret_msg":"OK","ext_code":"","ext_info":"","result":[{"name":"BTCUSD","base_currency":"BTC","quote_currency":"USD","price_scale":2,"taker_fee":"0.00075","maker_fee":"-0.00025","leverage_filter":{"min_leverage":1,"max_leverage":100,"leverage_step":"0.01"},"price_filter":{"min_price":"0.5","max_price":"999999.5","tick_size":"0.5"},"lot_size_filter":{"max_trading_qty":1000000,"min_trading_qty":1,"qty_step":1}},{"name":"ETHUSD","base_currency":"ETH","quote_currency":"USD","price_scale":2,"taker_fee":"0.00075","maker_fee":"-0.00025","leverage_filter":{"min_leverage":1,"max_leverage":50,"leverage_step":"0.01"},"price_filter":{"min_price":"0.05","max_price":"99999.95","tick_size":"0.05"},"lot_size_filter":{"max_trading_qty":1000000,"min_trading_qty":1,"qty_step":1}},{"name":"EOSUSD","base_currency":"EOS","quote_currency":"USD","price_scale":3,"taker_fee":"0.00075","maker_fee":"-0.00025","leverage_filter":{"min_leverage":1,"max_leverage":50,"leverage_step":"0.01"},"price_filter":{"min_price":"0.001","max_price":"1999.999","tick_size":"0.001"},"lot_size_filter":{"max_trading_qty":1000000,"min_trading_qty":1,"qty_step":1}},{"name":"XRPUSD","base_currency":"XRP","quote_currency":"USD","price_scale":4,"taker_fee":"0.00075","maker_fee":"-0.00025","leverage_filter":{"min_leverage":1,"max_leverage":50,"leverage_step":"0.01"},"price_filter":{"min_price":"0.0001","max_price":"199.9999","tick_size":"0.0001"},"lot_size_filter":{"max_trading_qty":1000000,"min_trading_qty":1,"qty_step":1}},{"name":"BTCUSDT","base_currency":"BTC","quote_currency":"USDT","price_scale":2,"taker_fee":"0.00075","maker_fee":"-0.00025","leverage_filter":{"min_leverage":1,"max_leverage":100,"leverage_step":"0.01"},"price_filter":{"min_price":"0.5","max_price":"999999.5","tick_size":"0.5"},"lot_size_filter":{"max_trading_qty":100,"min_trading_qty":0.001,"qty_step":0.001}}],"time_now":"1591736501.701404"}

fetch:
 bybit GET https://api-testnet.bybit.com/private/linear/trade/execution/list?api_key=xxx&limit=100&recv_window=5000&start_time=1591635962000&symbol=BTCUSDT&timestamp=1591736501911&sign=cfefaff576c68d68796706134cbe63f56128c5e786682df359141693c8485de0
Request:
 {}
 undefined

handleRestResponse:
 bybit GET https://api-testnet.bybit.com/private/linear/trade/execution/list?api_key=xxx&limit=100&recv_window=5000&start_time=1591635962000&symbol=BTCUSDT&timestamp=1591736501911&sign=cfefaff576c68d68796706134cbe63f56128c5e786682df359141693c8485de0 200 OK
Response:
 {
  'Cache-Control': 'max-age=0, no-cache, no-store',
  Connection: 'keep-alive',
  'Content-Length': '1137',
  'Content-Type': 'application/json; charset=utf-8',
  Date: 'Tue, 09 Jun 2020 21:01:42 GMT',
  Expires: 'Tue, 09 Jun 2020 21:01:42 GMT',
  Pragma: 'no-cache'
}
 {"ret_code":0,"ret_msg":"OK","ext_code":"","ext_info":"","result":{"current_page":1,"data":[{"order_id":"b59418ec-14d4-4ef9-b9f4-721d5d576974","order_link_id":"","side":"Sell","symbol":"BTCUSDT","exec_id":"0327284d-faec-5191-bd89-acc5b4fafda9","price":0.5,"order_price":0.5,"order_qty":0.01,"order_type":"Market","fee_rate":0.00075,"exec_price":9709.5,"exec_type":"Trade","exec_qty":0.01,"exec_fee":0.07282125,"exec_value":97.095,"leaves_qty":0,"closed_size":0.01,"last_liquidity_ind":"RemovedLiquidity","trade_time":1591648052,"trade_time_ms":1591648052861},{"order_id":"88858786-b56f-4f40-8624-c8d00227ce5f","order_link_id":"","side":"Buy","symbol":"BTCUSDT","exec_id":"ea872a29-fc87-51c7-b381-9cdb4a4f8de9","price":9707.5,"order_price":9707.5,"order_qty":0.01,"order_type":"Limit","fee_rate":0.00075,"exec_price":9700.5,"exec_type":"Trade","exec_qty":0.01,"exec_fee":0.07275375,"exec_value":97.005,"leaves_qty":0,"closed_size":0,"last_liquidity_ind":"RemovedLiquidity","trade_time":1591642216,"trade_time_ms":1591642216543}]},"time_now":"1591736501.979264","rate_limit_status":119,"rate_limit_reset_ms":1591736501974,"rate_limit":120}

[]
```

**Comment by kroitor** [comment]
Hi! Thx for reporting! I will look into it and will add a fix shortly.

**Comment by kroitor** [comment]
The empty response is fixed in CCXT 1.28.60+. Let us know if you have further difficulties with it. We will appreciate if you report back whether this resolves the issue on your side or not after upgrading to the most recent version of CCXT.

On your second question:

> 

> Another unrelated small question, for creating an order the parameters `close_on_trigger` and `reduce_only` are mandatory for bybit, is there any reason to not put a default value?

Yes, the reason to not put default values for those two params there: Bybit does not define any default value for those params in the API docs, so there's no way to decide which default value would be better for the majority of the users.

Let us know if that does not help.

**Comment by davidchieregato** [comment]
Thank you, I confirm it's fixed, now it's working.

For what I can undestand those params are probably generally false for most of the users to get a behavior similar to other exchanges (e.g. in Binance reduce_only is false by default). The particular cases are when a stoploss is inserted that having those params may get useful.

But I see the general point.
